sonar/sonar.go

```diff
@@ -183,14 +185,15 @@ func (s *Sonar) HandleRepoCommit(ctx context.Context, evt *comatproto.SyncSubscr
 	}
 
 	// Parse time from the event time string
-	t, err := time.Parse(time.RFC3339, evt.Time)
+	evtCreatedAt, err := time.Parse(time.RFC3339, evt.Time)
```

Comment:
does RFC3339 work here? I wrote `util.ParseTimestamp` to deal with the large variety of different formats ive seen on records

Reply:
Oh, I see you know about that. carry on

Reply:
Oh I started using a better thing like here: https://github.com/ericvolp12/bsky-experiments/blob/main/pkg/consumer/consumer.go#L256

https://github.com/araddon/dateparse/ - Faster than what we're currently doing i.e. "shotgun parsing"

For Sonar I think it makes sense for me to update to use that library so I'll patch that today.